varcode/vcf.py

```diff
@@ -230,15 +231,15 @@ def sample_info_parser(unparsed_sample_info_strings, format_string):
 
     return dataframes_to_variant_collection(
         df_iterator,
+        source_path=path,
         info_parser=handle.vcf_reader._parse_info if include_info else None,
         only_passing=only_passing,
         max_variants=max_variants,
         sample_names=handle.vcf_reader.samples if include_info else None,
         sample_info_parser=sample_info_parser,
         variant_kwargs={
             'ensembl': genome,
-            'allow_extended_nucleotides': allow_extended_nucleotides},
-        variant_collection_kwargs={"path": path})
+            'allow_extended_nucleotides': allow_extended_nucleotides})
```

Comment:
Not any more, `dataframes_to_variant_collection` populates metadata & sources on its own.